### PR TITLE
Added 2020 Call for Volunteers Blog

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
* **Please check if the PR fulfills these requirements**

- [ ] Screenshots for changes (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Blog Content

* **Other information**:
I've added a new blog post for the Call for Volunteers. It's a little lengthier than the previous year blogs, but it's still about a 5 min read. 
Please review. 

CC: @vipulgupta2048

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
Also tagging @Pradhvan for review. 
I feel we can add workgroup leaders names for direct contacts + some detail about stack that would be required.

**Comment by Pradhvan** [comment]
@Sc0rpi0n101 great writeup, just one thing if you could mention about volunteers meet that would be great. The weekly meet thing could come up in `What’s required of you?` with something like "All the workgroup leaders and volunteers have a weekly meeting to brief about the progress in their respective workgroup so that everyone's on the same page."

Just my suggestion feel free to improvise what you seem fit. 😄

**Comment by Sc0rpi0n101** [comment]
Sure

> I feel we can add workgroup leaders usernames for direct contacts on Zulip + some detail about stack that would be required.

Good idea. Can you give me all these details so I can add them?

**Comment by Pradhvan** [comment]
@Sc0rpi0n101 Here is the list https://github.com/pythonindia/inpycon2020-tasks You can mention names of chairs(C). Social Media has no chair as of now so feel free to put any name from the three in the list. 😄

**Comment by Sc0rpi0n101** [comment]
How's this? and is there anything else I should add?

**Comment by Pradhvan** [comment]
LGTM 👍

File: content/blog/2020-volunteering.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Date: 2020-06-30 22:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-call-for-volunteers
+
+Have you ever wished you could contribute or give back to the Python Community? Well, here’s an awesome opportunity for you to do it at PyCon. 
+
+[PyCon India](https://in.pycon.org/2020/) is a grass-roots, community-driven event. What that means is that various communities across the nation and people that love python come together and help in every way they can, to run the event. The conference is operated entirely by volunteers from planning it, finding the speakers, to handling the event successfully.
+
+<!-- PELICAN_END_SUMMARY -->
```
[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply content/blog/2020-volunteering.md:11]:
```suggestion

<!-- PELICAN_END_SUMMARY -->

[PyCon India](https://in.pycon.org/2020/) is a grass-roots, volunteer-driven conference. What that means is every year for the past 12 years Python communities across the nation have come together to help plan, organize, manage, and run the conference. The conference is operated entirely by volunteers from planning to finding the speakers to managing call for proposals to handling the mega event successfully without fail.
```

@Sc0rpi0n101 Let's give depth to what PyCon India means.

File: content/blog/2020-volunteering.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+## What you need to know. 
+
+Before you embark on a journey helping the community make the conference even more amazing, there are a few things you should know:
+
+* There is a lot of work that goes into a conference, and all of it is done by the volunteers. So, naturally, volunteering requires a lot of time and effort, and some activities like handling social media and registrations, a lot more than others. So, make sure you can allocate sufficient time to your volunteering efforts.
```
Comment (vipulgupta2048) [inline content/blog/2020-volunteering.md:17]: We actually don't need any help with registrations. 

 handling social media --> Running outreach and social media, building content, finding speakers

File: content/blog/2020-volunteering.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+Title: Making PyCon India even more amazing with your help
```
[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply content/blog/2020-volunteering.md:1]: Actually scratch that, we need a better title though.

[the remark this answers is not in the document]
Comment (Sc0rpi0n101) [reply content/blog/2020-volunteering.md:1]: How about `Help us make PyCon India even more amazing`?

**Comment by Sc0rpi0n101** [comment]
> Everything else looks good. Please make the changes suggested in the start.

I've already added the workgroup leaders. Can you give me some details about the required tech stacks?

File: content/blog/2020-volunteering.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply content/blog/2020-volunteering.md:1]: It has the helpful vibe, but no call to action.

**Comment by vipulgupta2048** [comment]
I think its fine with the tech stacks you have added.

**Comment by vipulgupta2048** [comment]
Thanks for the blog @Sc0rpi0n101 
@palnabarun if you can deploy thanks!

**Pull request merged**